hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HoodieHiveClient.java

```diff
@@ -244,7 +281,7 @@ public Option<String> getLastReplicatedTime(String tableName) {
 
   public void updateLastReplicatedTimeStamp(String tableName, String timeStamp) {
     if (!activeTimeline.filterCompletedInstants().getInstants()
-            .anyMatch(i -> i.getTimestamp().equals(timeStamp))) {
```

Comment:
can you please revert unintented changes. would be easy to review just the required change

Reply:
ok

Reply:
done